packages/instantsearch.js/src/connectors/toggle-refinement/connectToggleRefinement.ts

```diff
@@ -266,7 +273,9 @@ const connectToggleRefinement: ToggleRefinementConnector =
               });
```

Comment:
I think this code can probably be shared between createURL + refine right? (maybe doesn't need to be done in this PR if it's too complex)